TouchPortalAPI/tools.py

```diff
@@ -80,15 +80,25 @@ def updateCheck(name, repository):
             raise ValueError(f'No tags found in repository: {baselink}')
         else:
             raise ValueError(f'Invalid repository URL or response: {baselink}')
-            
-            
+                    
+    @staticmethod        
     def nested_conversion(value):
         """
-        Returns a dictionary back as a 'Class'
-     
-        - This allows for easy access to nested dictionary values
+        Convert dictionary to object for easier access data.
+        Examples
+            `data = {"car": {"year": 2008, "name": "Tesla"}}` # try get name
+
+            instead of this `data['car']['name']` you can get name this way
+
+            data = Tools.nested_conversion(data)
+
+            `data.car.name`
+
+        Args:
+            `value`: any dictionary
+        Raises:
+            `ValueError`: If value is any other then type dict
```

Comment:
Hmmm well it doesn't actually raise any error...  😃    But maybe it should... anyone expecting an object from an array will get an error anyway, at least if this throws it should be more obvious why.

Reply:
I was hopping maybe you can figure out because this It always raises an error says it's not a dict even though it is I am pretty confused 

```py
@staticmethod        
    def nested_conversion(value):
        """
        Convert dictionary to object for easier access data.
        Examples
            `data = {"car": {"year": 2008, "name": "Tesla"}}` # try get name

            instead of this `data['car']['name']` you can get name this way

            data = Tools.nested_conversion(data)

            `data.car.name`

        Args:
            `value`: any dictionary
        Raises:
            `ValueError`: If value is any other then type dict
        """
        if not isinstance(value, dict):
            raise ValueError(f"nested_conversion() requires an dict, got {type(value)} instead.")
        return SimpleNamespace(**{key: Tools.nested_conversion(value) for key, value in value.items()})
```

Reply:
Ah, my mistake, it _has to_ return a `dict` type when it recurses into itself.  Otherwise it throws that error after a few recursive loops.

I guess just remove the  docs part about "Raises" and call it good!  😄